resolwe/flow/tests/test_executors.py

```diff
@@ -344,31 +347,34 @@ def test_slug_colision(self):
         This checks for a bug in permission architecture that caused exception
         due to missing permission_group when slug colision occured.
         """
-        process = Process.objects.filter(slug="test-min").latest()
+        def pre_save(instance, add):
+            current_slug = slugs.pop(0)
+            if current_slug == IntegrityError:
```

Comment:
It would be a bit more general to check whether the current slug is a subclass of `Exception`.

Reply:
Sure.